src/components/utrecht/Heading1.tsx

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license EUPL-1.2
+ * Copyright (c) 2021 Robbert Broersma
+ */
+
+import clsx from "clsx";
+import { ForwardedRef, forwardRef, HTMLAttributes, PropsWithChildren } from "react";
+
+export type Heading1Props = HTMLAttributes<HTMLHeadingElement>;
```

Comment:
I would suggest not exporting this type as a default api. Probably no need to re-use it anywhere and Typescript will already suggest these properties when using the component without exporting it.